scalding/src/main/scala/com/matthewrathbone/scalding/Main.scala

```diff
@@ -0,0 +1,35 @@
+package main.scala.com.matthewrathbone.scalding
+
+import com.twitter.scalding._
+import cascading.tuple.TupleEntry
+
+class Main ( args: Args ) extends Job( args )
+  {
+  
+    val inputFields = 'line
+	val users = ( 'id, 'email, 'language, 'location)
+  	val transactions = ( 'transaction_id, 'product_id, 'user_id, 'purchase_amount, 'item_description);
+	
+	val input1 = TextLine( args( "input1" ) )
```

Comment:
could you add the test files to the codebase so people can just run this and have it work without having to make them? Maybe put them somewhere like `data/users` and `data/transactions`?